lib/tlscontext.h

```diff
@@ -44,6 +44,16 @@ typedef enum
   TVM_REQUIRED=0x0020,
 } TLSVerifyMode;
 
+typedef enum
+{
+  TSO_NONE,
+  TSO_NOSSLv2=0x0001,
+  TSO_NOSSLv3=0x0002,
+  TSO_NOTLSv1=0x0004,
+  TSO_NOTLSv11=0x0008,
+  TSO_NOTLSv12=0x0010,
+} TLSSslOptions;
+
```

Comment:
hi,

I told you IRL that this is not about consistency: it's about correctness.
(a legacy code will always contain some legacy code ;-) That doesn't mean we should always follow it
and doesn't mean that it cannot be changed... )

I think 2nd option in your list would be fine with an additional note:
If we have a chance for writing a not-legacy code, we should have to do that, IMHO. 

L.